_data/sites.json

```diff
@@ -5522,6 +5522,16 @@
         ]
     },
 
+    {
+        "name": "Lutris",
+        "url": "https://lutris.net",
+        "notes": "After logging in, click on your profile picture in the top right. Then, click on 'Edit Profile,' once on that page, click 'Delete my account.'",
```

Comment:
Do you have a link to the profile page directly? That would make it even easier for visitors of our service to delete it.

Reply:
As far as I know, that's the closest link you can get to the page. The page to actually delete your account would look like this: `https://lutris.net/user/usernamehere/edit`. From what I can tell, there wouldn't be a way to directly link it without knowing the username of the person trying to delete their account.